<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaDeEdicao</class>
 <widget class="QDialog" name="telaDeEdicao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>461</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Logradouro:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditLogradouro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Numero:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNumero"/>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>551</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabTelefone">
    <attribute name="title">
     <string>Telefone</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetListaTelefone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>DDI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DDD</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefone</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>534</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>DDI:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDDI"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>DDD:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDDD"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>NÃºmero:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNumeroTel"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_incluirMais">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabEmail">
    <attribute name="title">
     <string>Email</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetEmail">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>441</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditEmail"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_incluirMaisEmail">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>461</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNome"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonEditar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancelar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditID">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
